tests/topotests/bgp_bmp/test_bgp_bmp_2.py

```diff
@@ -252,6 +252,169 @@ def test_peer_down():
     assert success, "Checking the updated prefixes has been failed !."
 
 
+def test_bgp_instance_flapping():
+    """
+    Checking for BGP loc-rib up messages
+    """
+    tgen = get_topogen()
+
+    # create flapping at BMP
+    tgen.net["r1vrf"].cmd("ip link set dev vrf1 down")
+
+    peers = ["0.0.0.0"]
+    logger.info("checking for BMP peer down LOC-RIB message.")
+    test_func = partial(
+        bmp_check_for_peer_message,
+        peers,
+        "peer down",
+        tgen.gears["bmp1vrf"],
+        os.path.join(tgen.logdir, "bmp1vrf", "bmp.log"),
```

Comment:
What would this path look like if I did?

```
cp tests/topotests/bgp_bmp/test_bgp_bmp_2.py tests/topotests/bgp_bmp/test_bgp_bmp_3.py
pytest -n 2 -s
```
Won't test_bgp_bmp_2 override test_bgp_bmp_3 logs or vice-versa?

Reply:
> What would this path look like if I did?
> 
> ```
> cp tests/topotests/bgp_bmp/test_bgp_bmp_2.py tests/topotests/bgp_bmp/test_bgp_bmp_3.py
> pytest -n 2 -s
> ```
> 
> Won't test_bgp_bmp_2 override test_bgp_bmp_3 logs or vice-versa?

mmm, if the user uses 'bmp1vrf' in `test_bgp_bmp_3.py`, then the files `/tmp/topotests/bgp_bmp.test_bgp_bmp_3/bmp1vrf/...` will be used. It should not erase `/tmp/topotests/bgp_bmp.test_bgp_bmp_2/bmp1vrf/...`

Reply:
@mxyns , you already partially reviewed the original pull request. please review the extensions proposed.